media/css/firefox/pocket.scss

```diff
@@ -9,7 +9,6 @@ $image-path: '/media/protocol/img';
 @import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
 @import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-pocket';
 @import '~@mozilla-protocol/core/protocol/css/components/billboard';
-@import '~@mozilla-protocol/core/protocol/css/components/hero';
```

Comment:
**issue (blocking)**: there are still `mzp-c-hero` classes on the bottom section, and the section is stacked instead of side-by-side

<img width="1234" alt="pocket-hero" src="https://user-images.githubusercontent.com/19650432/212358860-c4892e63-9f86-4b52-9ed7-cce95a19589b.png">

vs prod

<img width="1094" alt="pocket-prod" src="https://user-images.githubusercontent.com/19650432/212359279-9a3dd967-8e7d-468c-a09d-6d864eb331ce.png">

**suggestion**: replace with split?